.github/workflows/test_publish.yml

```diff
@@ -0,0 +1,70 @@
+name: Integration Test when Publishing to External Data Warehouses
+
+on:
+  pull_request:
+    types: 
+    - closed
```

Comment:
why run on closed pull requests? This will run the job twice if the PR is merged to master.

Reply:
There is also a condition for PR being on `master`. So, this runs when a PR on master is closed (which means the PR is merged to `master`). I don't get why it runs twice.

Reply:
oh I see, sorry I confused it with being run on push to master (not PR to master). I think we tend to use `push` instead of `pull_request` in some of our other configs, so something like this:

```yaml
on:
  push:
    branches: [master]
    paths: [renku/core/commands/providers/*.py]
```

Reply:
Using `push` is simpler actually 🤦‍♂